File: Java Klassen/src/model/plan/BuildCity.java
package model.plan;

import model.CityBuildPlace;

/**
 * Klasse, die die Aktion des Bauens von Städten darstellt
 * 
 * @author tjark
 */
public class BuildCity extends Action {
	
	private int row, column;
	
	public BuildCity() {}

	public BuildCity(CityBuildPlace randomCityPlace) {
		this.row = randomCityPlace.row;
		this.column = randomCityPlace.column;
	}

	public int getRow() {
		return row;
	}

	public void setRow(int row) {
		this.row = row;
	}

	public int getColumn() {
		return column;
	}

	public void setColumn(int column) {
		this.column = column;
	}

}

File: Unity Projekt/Assets/Scripts/CBR.Plan/BuildCity.cs
using Assets.Scripts.Model;
using System.Runtime.Serialization;

namespace Assets.Scripts.CBR.Plan
{        
    
    /**
    * Klasse, die die Aktion das Bauen von Straßen darstellt
    */
    [DataContract]
    public class BuildCity : Action
    {

        /**
        * Attribut zur Speicherung, in welcher Zeile gebaut werden soll 
        */
        [DataMember]
        public int row;

        /**
         * Attribut zur Speicherung, in welcher Spalte gebaut werden soll 
         */
        [DataMember]
        public int column;

        public BuildCity(int row, int column) : base("BuildCity")
        {
            this.row = row;
            this.column = column;
        }


        public BuildCity() : base("BuildCity")
        {
        }
    }
}
